solutions/bronze/usaco-941.mdx

```diff
@@ -5,15 +5,33 @@ title: Cow Evolution
 author: Jesse Choe, Ryan Chou, Chuyang Wang, Vivian Han
 ---
 
-## Solution
-[Official Analysis (C++)](http://www.usaco.org/current/data/sol_evolution_bronze_open19.html)
-
 ## Video Solution
+
 <Youtube id="4YTijr0BbrA" />
 
+<Spoiler title="Hint">
+
+Try to construct a minimal set of cows that can't form a valid evolutionary tree.
+What's a defining trait about this bad set that you can check through code?
```

Comment:
```suggestion
How can we generalize to different trees?
```

Reply:
i think telling the reader to check it in code implies that this minimal set is the only thing to take care of...

(would it be better to have the reader try to prove it themselves?)